lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -24,9 +24,11 @@
 
 #include <pxr/usdImaging/usdImaging/delegate.h>
 
+#ifdef MAYA_HAS_DISPLAY_LAYER_API
 #include <maya/MFnDisplayLayer.h>
 #include <maya/MFnDisplayLayerManager.h>
 #include <maya/MObjectArray.h>
+#endif
```

Comment:
According to our coding standards, conditional includes come last after all other includes. I'm not sure if clang-format checks this. So line 32 should really come before this ifdef